core/src/util/jobQueue.cpp

```diff
@@ -0,0 +1,40 @@
+#include "jobQueue.h"
+#include <mutex>
+
+namespace Tangram {
+
+std::unordered_map<JobQueue::ThreadId, std::vector<JobQueue::Job>> JobQueue::jobsForThreads;
+std::mutex jobMutex;
```

Comment:
Static member vs global variable!
Purpose is to make read/writes to `jobsForThreads` synchronous. Both will achieve the same thing.